kolibri/core/assets/src/mixins/theme.js

```diff
@@ -0,0 +1,8 @@
+import { mapGetters } from 'vuex';
+import themeModule, { nameSpace } from '../state/modules/theme';
+
+export default {
+  computed: {
+    ...mapGetters(nameSpace, Object.keys(themeModule.getters)),
+  },
+};
```

Comment:
mind adding some comments explaining what's going on with the `nameSpace` stuff? It's not immediately obvious to me how it works